superset-frontend/plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -118,6 +118,7 @@ const all_columns: typeof sharedControls.groupby = {
   }),
   visibility: isRawMode,
   resetOnHide: false,
+  showSelectAllButton: true,
```

Comment:
```suggestion
  showSelectAll: true,
```

I think it's okay to call this prop `showSelectAll` in case we want to change how the "select all" trigger looks like.